dig.php

```diff
@@ -4,242 +4,43 @@
     die('This command can only be used in CLI mode.');
 }
 
-function startDevelopmentServer($pidfile)
-{
-    $cmd = 'php -S localhost:8082';
-    $outputfile = '/dev/null';
-    echo 'location: ' . getcwd() . "\n";
-    shell_exec(sprintf("%s > %s 2>&1 & echo $! >> %s", $cmd, $outputfile, $pidfile));
-    sleep(1);
-}
-
-function killDevelopmentServer($pidfile)
-{
-    if (file_exists($pidfile)) {
-        $pids = file($pidfile);
-        foreach ($pids as $pid) {
-            shell_exec('kill -9 ' . $pid);
-        }
-        if (unlink($pidfile)) {
-            echo "Successfully deleted " . $pidfile;
-        } else {
-            echo "Problem deleting " . $pidfile;
-        }
-    }
-}
-
 date_default_timezone_set('Europe/Paris');
 mb_internal_encoding('UTF-8');
 
-// Manage CTRL+C and other script interruptions
-declare(ticks = 1); // how often to check for signals
-// This function will process sent signals
-function sig_handler($signo)
-{
-    global $git_mozorg;
-    global $svn_misc;
-    if ($signo == SIGTERM || $signo == SIGHUP || $signo == SIGINT) {
-        if ($signo != 15) {
-            print "Received signal $signo and will exit now!\n";
-        } else {
-            print "Operations finished\n";
-        }
-        // Cleanup
-        chdir($svn_misc);
-        exec('svn cleanup');
-        exec('pkill php');
-        exit;
-    }
-}
-
-// These define the signal handling
-pcntl_signal(SIGTERM, 'sig_handler');
-pcntl_signal(SIGHUP,  'sig_handler');
-pcntl_signal(SIGINT,  'sig_handler');
-
-// Repositories to loop for date in
-$app             = realpath(__DIR__);
-$pid_file        = $app . '/pidfile.txt';
-$data_path       = $app   . '/logs/data.json';
-$repos           = $app   . '/repos';
-$git_mozorg      = $repos . '/mozillaorg/';
-$svn_misc        = $repos . '/l10n-misc';
-$git_langchecker = $repos . '/langchecker';
-$git_stores      = $repos . '/appstores';
+// Declare paths and time
+$app               = realpath(__DIR__);
+$archive_data_path = $app . '/archive.json';
+$live_data_path    = $app . '/logs/live.json';
+$today             = new DateTime('now');
 
-// Create our data structure
-$update_repos = true;
-
-if (! is_dir($git_mozorg)) {
-    print "Checking our mozilla.org git repository\n";
-    mkdir($repos . '/mozillaorg/');
-    chdir($repos);
-    exec('git clone https://github.com/mozilla-l10n/www.mozilla.org/ mozillaorg');
-} elseif ($update_repos) {
-    chdir($git_mozorg);
-    print "Updating the mozilla.org git repository\n";
-    exec('git checkout master');
-    exec('git pull origin master');
-}
-
-if (! is_dir($svn_misc)) {
-    chdir($repos);
-    print "Checking our l10n-misc svn repository\n";
-    exec('svn co https://svn.mozilla.org/projects/l10n-misc/trunk l10n-misc');
+if (! file_exists($live_data_path)) {
+    file_put_contents($live_data_path, json_encode([]));
```

Comment:
See comment on the original version of the PR, I find this quite confusing: the file is missing, so we write an empty file, then read it instead of just setting the data as an empty array.